db/migrate/20210325033949_rename_city_to_address_in_groups.rb

```diff
@@ -0,0 +1,5 @@
+class RenameCityToAddressInGroups < ActiveRecord::Migration[6.1]
```

Comment:
remove this migration

Reply:
If I remove that I'll need to change the new migration to

Reply:
If I remove that migration I'll need to change the new migration to ```rename_column :groups, :city, :train_station```